sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -23,6 +23,60 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     using SignedIdentifier = Blobs::Models::SignedIdentifier;
     using FileQueryArrowField = Blobs::Models::BlobQueryArrowField;
     using FileQueryArrowFieldType = Blobs::Models::BlobQueryArrowFieldType;
+
+    /**
+     * @brief An access control object.
+     */
+    struct Acl final
```

Comment:
A minor nit: An "acl" is an "access control list" - in other words, it's a collection of objects. This appears to be an individual object.

The objects which make up an ACL are called "Access Control Entries" or ACE.

So I believe that pedantically, this should be an `Ace`, not an `Acl`.

Reply:
This is a valuable suggestion, I also think `Ace` may be better,but this was moved from somewhere else and would be a breaking change to change its name.